board-printer.js

```diff
@@ -15,7 +21,24 @@
         =================
     Test your function by calling it with an example tic-tac-toe board.
 */
+
 export function printBoard(board) {
+  board.forEach((row) => {
+    printRow(row);
+    console.log(`==========`)
+  });
+}
+
+function printRow(row) {
+    let newRow = row.map((el) => {
+        if (el === "_") {
+            return " "
+        } else {
+            return el
+        }
```

Comment:
you could condense this down to a ternary operator to make it cleaner - `return el === "_" ? " " : el`. In general, anytime you have the pattern:
```
if (...) {
   return x
} else {
   return y
}
```
it can be reduced to a ternary expression